src/react-native/06-building-custom-components/building-custom-components.md

```diff
@@ -2,23 +2,66 @@
 @parent learn-react-native 6
 @outline 3
 
-@description TODO
+@description Learn about components, the core building blocks of every React application.
 
 @body
 
 ## Overview
 
 In this section, you will:
 
-- TODO
+- Learn the basics of creating components in React.
+- Discover how components are structured.
+- Review how React components are (fundamentally) functions.
+- Understand what props are and how they work in React.
```

Comment:
Avoid terms like “learn” and “discover,” and instead say what they will be doing. The concept headers are phrased like “Using component props” so translate that to “Use component props.”